niworkflows/viz/utils.py

```diff
@@ -87,12 +88,10 @@ def _run(args, input=None, stdout=None, stderr=None, shell=False, check=False,
 
 
 def robust_set_limits(data, plot_params):
-    vmin = np.percentile(data, 15)
-    if plot_params.get('vmin', None) is None:
-        plot_params['vmin'] = vmin
-    if plot_params.get('vmax', None) is None:
-        plot_params['vmax'] = np.percentile(data[data > vmin], 99.8)
-
+    plot_params['vmin'] = plot_params.get(
+        'vmin', np.percentile(data, 15))
```

Comment:
Wouldn't this change how a lot of figures are plotter? Have you checked if there are no side effects?

Reply:
I haven't seen any problem, but the logic of this code should be equivalent to the former.